packages/core/src/http-client/http-client-types.ts

```diff
@@ -9,6 +9,25 @@ export interface DestinationHttpRequestConfig {
   httpsAgent?: http.Agent;
 }
 
+type Method =
+  | 'get'
```

Comment:
is there a more elegant way around the "CAP" "noncap" maintanance?

Reply:
Can you elaborate on what you mean with "CAP" and "noncap"?

Reply:
I also don't understand.

Reply:
I mean that we have 'GET' and 'get' as potential values for the method. If we use this only internally we should stick to one. If this is exposed and we want to be lenient to the user so that one can use 'GET' or 'get'. I was then wondering if there is a case insensitive union type or enum, but there [is not](https://stackoverflow.com/questions/43677527/typescript-type-ignore-case).

Reply:
Aaah! CAP = capital. The enum that this union type would supersede was originally inspired by Axios. Either way, removing either the lowercase or the uppercase members would be a breaking change.

Reply:
Up to now we only support uppercase, therefore removing lowercase would not be a breaking change. But again, I don't see the problem also supporting lowercase. I would actually vote for us using lowercase in our code too (we are not consistent here).

Reply:
Based on your comment below, I will resolve this one.